### PR TITLE
Fixed a typo in syntax

### PR DESCRIPTION
**Pull request by frol**
@mikedotexe Git history suggests that you are working on the READMEs. I would like to point out that it is unclear what are the next steps after you open the Gitpod example (it opens the README-Gitpod file which has very little context) without prior reading and completely understanding the flow described in the main README. I would suggest having a guide in the README-Gitpod. WDYT?

### CODE REVIEW

**Comment by frol** [comment]
I had just noticed your PR https://github.com/near-examples/guest-book/pull/41 and it looks fantastic!

**Comment by frol** [comment]
I opened the example and I immediately wanted to know:

1. What is this all about? (starting with a link to some intro into Gitpod itself, and following with an intro about NEAR and the example itself in plain English [ideally, without "smart contract" in the explanation])
2. What can I do next? (I argue that examples should first teach people WHAT, and only then HOW)

If we want to improve it even further, I think, we should engage users to modify the code and play with it even further. To do that, I suggest we have some code snippets commented out and accompanied with explanations on what this additional code can achieve.

**Comment by mikedotexe** [comment]
That's a great idea. I'll bring this up in our DevX Huddle on Tuesday
